db/migrate/20190502185613_add_study_area_column_to_transportation_analysis.rb

```diff
@@ -0,0 +1,5 @@
+class AddStudyAreaColumnToTransportationAnalysis < ActiveRecord::Migration[5.2]
+  def change
+    add_column :transportation_analyses, :jtw_study_area, :geometry, limit: {:srid=>4326, :type=>"point"}, null: false
```

Comment:
Should this column be called `jtw_study_centroid` ? `_area` makes me think it's a polygon.

Reply:
Or `jtw_study_area_centroid`

Reply:
Totally